### ISSUE TITLE
Q126 Extra brackets

### ISSUE THREAD

**Issue by contributorpw** [lydiahallie/javascript-questions#279]
The Q126 has extra brackets

![1574833722601](https://user-images.githubusercontent.com/5027939/69697155-d0d56f00-10f2-11ea-814b-e01c487de775.png)

**Comment by tshinnic** [comment]
First MDN example - without extra braces - works in Chrome 81 (canary):
```
> const number = 123456.789;
> console.log(new Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' }).format(number));
  123.456,79 €
```
Add braces in and an error message results:
```
> console.log(new Intl.NumberFormat({'de-DE', { style: 'currency', currency: 'EUR' }}).format(number));
  Uncaught SyntaxError: Unexpected token ','
```

**Comment by contributorpw** [comment]
@tshinnic  I think it's a typo. Please check #281

**Comment by tshinnic** [comment]
Yes, I was wanting to use an example from another source, to confirm the extra braces were unneeded/invalid, in agreement with your fix.

**Comment by jakeherp** [comment]
Merged #281
